### ISSUE TITLE
InitializeUser fails

### ISSUE THREAD

**Issue by creategui** [deviceplane/deviceplane#86]
[InitializeUser](https://github.com/deviceplane/deviceplane/blob/9ea53e653b89b7064ca9f9f1a35e6acf85a3d29f/pkg/controller/store/mysql/store.go#L148) fails when registering/confirming a new user because the [insert into users](https://github.com/deviceplane/deviceplane/blob/master/pkg/controller/store/mysql/sql.go#L3) does not include the column 'name' which is [set to not null](https://github.com/deviceplane/deviceplane/blob/master/pkg/controller/store/mysql/schema.sql#L49).

A quick fix to get past that is to alter the table and provide a default value:
ALTER TABLE users ALTER name SET DEFAULT 'Default Name';
or make it nullable:
ALTER TABLE users MODIFY name VARCHAR(100);

**Comment by dwebfan** [comment]
Not sure if the repo is still actively maintained or not. But I've fixed it in my fork https://github.com/dwebfan/deviceplane, in case you are interest.

**Comment by creategui** [comment]
> Not sure if the repo is still actively maintained or not. But I've fixed it in my fork https://github.com/dwebfan/deviceplane, in case you are interest.

No sure either, I have fixed it as well in my fork at https://github.com/creategui/deviceplane along with a couple other things to help with self-hosting.  I have it deployed at GCP in CloudRun connecting to a CloudSQL instance.

**Comment by dwebfan** [comment]
Nice to hear that! I'll pick your other change as well!

**Comment by drluckyspin** [comment]
Yes, deviceplane has been very quiet, but it is still going to be supported and in fact a bunch of new stuff coming. Watch this Space!

Also, we'd love those PRs submitted back please =)
